js/testing_1_2_3.js

```diff
@@ -0,0 +1,10 @@
+// https://www.codewars.com/kata/testing-1-2-3/train/javascript
+
+const number=function(array){
+  return array.map(function(el, i){
+     el = `${i+1}: ${el}`;
```

Comment:
What is the reason for reassigning this parameter?

Is it even necessary to save the result in a variable?